static/app/views/settings/organizationDeveloperSettings/permissionSelection.spec.jsx

```diff
@@ -27,74 +30,45 @@ describe('PermissionSelection', () => {
   });
 
   it('renders a row for each resource', () => {
-    expect(wrapper.find('SelectField[key="Project"]')).toBeDefined();
-    expect(wrapper.find('SelectField[key="Team"]')).toBeDefined();
-    expect(wrapper.find('SelectField[key="Release"]')).toBeDefined();
-    expect(wrapper.find('SelectField[key="Event"]')).toBeDefined();
-    expect(wrapper.find('SelectField[key="Organization"]')).toBeDefined();
-    expect(wrapper.find('SelectField[key="Member"]')).toBeDefined();
+    expect(screen.getByRole('textbox', {name: 'Project'})).toBeInTheDocument();
+    expect(screen.getByRole('textbox', {name: 'Team'})).toBeInTheDocument();
+    expect(screen.getByRole('textbox', {name: 'Release'})).toBeInTheDocument();
+    expect(screen.getByRole('textbox', {name: 'Issue & Event'})).toBeInTheDocument();
+    expect(screen.getByRole('textbox', {name: 'Organization'})).toBeInTheDocument();
+    expect(screen.getByRole('textbox', {name: 'Member'})).toBeInTheDocument();
   });
 
-  it('lists human readable permissions', () => {
-    const findOptions = name => {
-      openMenu(wrapper, {name: `${name}--permission`});
-      return wrapper
-        .find(`SelectField[name="${name}--permission"] Option`)
-        .map(o => o.text());
+  it('lists human readable permissions', async () => {
+    const expectOptions = async (name, options) => {
+      for (const option of options) {
+        await selectEvent.select(screen.getByRole('textbox', {name}), option);
+      }
     };
 
-    expect(findOptions('Project')).toEqual([
-      'No Access',
-      'Read',
-      'Read & Write',
-      'Admin',
-    ]);
-    expect(findOptions('Team')).toEqual(['No Access', 'Read', 'Read & Write', 'Admin']);
-    expect(findOptions('Release')).toEqual(['No Access', 'Admin']);
-    expect(findOptions('Event')).toEqual(['No Access', 'Read', 'Read & Write', 'Admin']);
-    expect(findOptions('Organization')).toEqual([
-      'No Access',
-      'Read',
-      'Read & Write',
-      'Admin',
-    ]);
-    expect(findOptions('Member')).toEqual(['No Access', 'Read', 'Read & Write', 'Admin']);
-  });
-
-  it('converts permission state to a list of raw scopes', () => {
-    const instance = wrapper.find('PermissionSelection').instance();
-    instance.setState({
-      permissions: {
-        Project: 'write',
-        Release: 'admin',
-        Organization: 'read',
-      },
-    });
-
-    expect(instance.permissionStateToList()).toEqual([
-      'project:read',
-      'project:write',
-      'project:releases',
-      'org:read',
-    ]);
```

Comment:
This test is _slightly_ lost, but not a big deal. Would be nice to check the API call mock for form submission and check that it gets the right scopes in this correct format